src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+import React from 'react';
+import { Tab } from '../../types/tab';
+
+interface TabsProps {
+  tabs: Tab[];
+  selectedTab: Tab;
+  handleTabClick: (tab: Tab) => void;
+}
+
+export const Tabs: React.FC<TabsProps> = (
+  { tabs, selectedTab, handleTabClick },
+) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => (
+            <li
+              key={tab.id}
+              className={selectedTab.id === tab.id ? 'is-active' : ''}
+              data-cy="Tab"
+            >
+              <a href={`#${tab.id}`} onClick={() => handleTabClick(tab)} data-cy="TabLink">
+                {tab.title}
+              </a>
+            </li>
+          ))}
+        </ul>
+      </div>
+      <div className="block" data-cy="TabContent">
```

Comment:
```suggestion
      <div className="block" data-cy="tab-content">
```
try to replace as in task description
<img width="797" alt="image" src="https://user-images.githubusercontent.com/53611910/230735836-c5863cd0-b2ca-4b77-828e-5dccb603c1f2.png">